### PR TITLE
Begin implementing first mockup

### PR DESCRIPTION
**Pull request by ConnorCimowsky**

### CODE REVIEW

File: mockups/base.html [hunk 1]
```diff
@@ -1,36 +0,0 @@
-<!DOCTYPE html>
```
Comment (Trinovantes) [inline mockups/base.html]: why delete this?

File: src-frontend/app/styles/base.less [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 @import url(constants.less);
-@import url(octicons.less);
 
 //------------------------------------------------------------------------------
 // General
 //------------------------------------------------------------------------------
 
 body{
-    background: #efefef;
+    background: #ececec;
```
Comment (Trinovantes) [inline src-frontend/app/styles/base.less]: this should be defined as a constant

File: src-frontend/app/styles/octicons.less [hunk 3]
```diff
@@ -1,4 +1,4 @@
-@octicons-font-path: "fonts";
+@octicons-font-path: ".";
```
Comment (Trinovantes) [inline src-frontend/app/styles/octicons.less]: fonts should be placed in a separate folder

File: src-frontend/config/environment.js [hunk 4]
```diff
@@ -22,11 +22,11 @@ module.exports = function(environment) {
...
     ENV.APP.API_HOST = 'http://localhost:8000';
     ENV.APP.API_NAMESPACE = 'api';
-    ENV.APP.LOG_RESOLVER = true;
-    ENV.APP.LOG_ACTIVE_GENERATION = true;
-    ENV.APP.LOG_TRANSITIONS = true;
-    ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
-    ENV.APP.LOG_VIEW_LOOKUPS = true;
+    ENV.APP.LOG_RESOLVER = false;
+    ENV.APP.LOG_ACTIVE_GENERATION = false;
+    ENV.APP.LOG_TRANSITIONS = false;
```
Comment (Trinovantes) [inline src-frontend/config/environment.js]: why disable logs in dev?

**Comment by Trinovantes** [comment]
I haven't looked too closely but there seems to be a bit of repeated css.

Also there's no need to merge in a commit that reverts another commit in the same PR

File: mockups/base.html (hunk 1 above)
Comment (ConnorCimowsky) [inline mockups/base.html]: I was trying to delete stuff as I port it over, but I agree I should just leave the mockups until I'm completely done. :sparkles:

File: src-frontend/app/styles/base.less (hunk 2 above)
Comment (ConnorCimowsky) [inline src-frontend/app/styles/base.less]: tru

File: src-frontend/app/styles/octicons.less (hunk 3 above)
Comment (ConnorCimowsky) [inline src-frontend/app/styles/octicons.less]: Forgot to move them back. Thanks. :+1:

File: src-frontend/config/environment.js (hunk 4 above)
Comment (ConnorCimowsky) [inline src-frontend/config/environment.js]: They print hundreds of lines for a simple page load, and don't provide any useful information. Not worth having them enabled all the time, IMO

**Comment by ConnorCimowsky** [comment]
I had done the merge/revert so that I would have that work recorded somewhere, but it looks like we're not going to take that approach anyway. Will rebase.

**Comment by ConnorCimowsky** [comment]
Re: CSS, maybe you can help me squash some of it later today. :innocent:

File: src-frontend/config/environment.js (hunk 4 above)
Comment (Trinovantes) [inline src-frontend/config/environment.js]: can you assign these with a local var (e.g. `var enableLogs = false`) so it'll be easier to toggle later on

**Comment by Trinovantes** [comment]
@Kapin what do you think about merging in revert commits? Also now I see how you got so many commits in master :p

**Comment by Kapin** [comment]
Avoid revert commits unless it's a specific revert. Within a PR do an interactive rebase :smile:

**Pull request merged**